project/src/components/pages/SearchPage.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react'
+import Header from '../HeaderLogOut'
+import Search from '../Search'
+import Nav from '../Nav'
+import UserBlock from "../Main";
+
+
+const SearchPage = () => (
```

Comment:
Your new components are very close to be the same. Again, the copy-paste.
Have to be fixed firstly!